com.unity.render-pipelines.universal/ShaderLibrary/Particles.hlsl

```diff
@@ -8,6 +8,37 @@
 #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/DeclareOpaqueTexture.hlsl"
 #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/ParticlesInstancing.hlsl"
 
+struct ParticleParams
+{
+    float4 positionWS;
+    float4 particleColor;
```

Comment:
is this vertex color? if so, maybe name it like that for better clarity.

Reply:
Good point! Renamed to vertexColor.